### PR TITLE
Add tensorflow license to transfer_learning

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

**Comment by yangw1234** [comment]
LGTM

File: pyzoo/zoo/examples/orca/learn/tf/transfer_learning/transfer_learning.py [hunk 1]
```diff
@@ -13,8 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+#
+# Copyright 2019 The TensorFlow Authors.
+#
+#@title Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (yangw1234) [inline pyzoo/zoo/examples/orca/learn/tf/transfer_learning/transfer_learning.py:19]: maybe we do not need `@title` ?

**Comment by leonardozcm** [comment]
hi @jason-dai, would you mind take a look?

File: pyzoo/zoo/examples/orca/learn/tf/transfer_learning/transfer_learning.py (hunk 1 above)
Comment (leonardozcm) [reply pyzoo/zoo/examples/orca/learn/tf/transfer_learning/transfer_learning.py]: > maybe we do not need `@title` ?

fixed

**Comment by yangw1234** [comment]
LGTM

**Comment by leonardozcm** [comment]
jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/4959/ @yangw1234

**Pull request merged**